src/DynamoRevit/DynamoRevit.cs

```diff
@@ -134,6 +137,22 @@ public Result Execute(ExternalCommandData commandData, ref string message, Eleme
             return Result.Succeeded;
         }
 
+        private static void OnApplicationIdle(object sender, IdlingEventArgs e)
```

Comment:
Adding a task to a scheduler need that the scheduler is initialized and the scheduler is not disposed when the operation happens. It has been done for fixing one defect previously:
https://github.com/DynamoDS/Dynamo/pull/3531

For your suggestion, I am thinking whether I can use the existing ExecuteOnIdleAsync for cases which happens when a valid scheduler is available. But I do not want to make my part of change depend on the RevitDynamoModel instance.

The point is that I am just subscribing/unsubscribing an event, why do I need to make it depend on more objects?